src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/note-dialog/note-dialog.component.ts

```diff
@@ -21,13 +21,17 @@ import { I18nService } from 'xforge-common/i18n.service';
 import { UserService } from 'xforge-common/user.service';
 import { objectId } from 'xforge-common/utils';
 import { FeatureFlagService } from 'xforge-common/feature-flags/feature-flag.service';
+import { isParatextRole } from 'realtime-server/lib/esm/scriptureforge/models/sf-project-role';
+import { UserDoc } from 'xforge-common/models/user-doc';
 import { NoteThreadDoc, defaultNoteThreadIcon } from '../../../core/models/note-thread-doc';
 import { SFProjectDoc } from '../../../core/models/sf-project-doc';
 import { SFProjectProfileDoc } from '../../../core/models/sf-project-profile-doc';
 import { SFProjectService } from '../../../core/sf-project.service';
-import { SFProjectUserConfigDoc } from '../../../core/models/sf-project-user-config-doc';
 import { TextDoc, TextDocId } from '../../../core/models/text-doc';
 import { canInsertNote, formatFontSizeToRems } from '../../../shared/utils';
+import { environment } from '../../../../environments/environment';
+
+const COMMENTER_LABEL_REGEX = /<p>\[.+\s-\s.+\]<\/p>\s*<p>(.+)<\/p>/s;
```

Comment:
This regular expression is greedy. For example, it will match:
```
<p>[Peter] I wrote this</p>
<p>[Test] This is a comment - read</p>
<p>[Peter]</p>
<p>Something</p>
```
You could make it non-greedy by replacing the wildcard with a negation, and anchoring it to the start, for example:
```
/^<p>\[[^\]]+\s-\s[^\]]+\]<\/p>\s*<p>(.+)<\/p>/s
```